### ISSUE TITLE
TO golang -- /api/1.3/cdns/health broken

### ISSUE THREAD

**Issue by dangogh** [apache/incubator-trafficcontrol#1730]
any /cdns/ sub-routes (other than the ":ID" route) will error with this:
```
{"alerts":[{"text":"getting CDNs response: pq: invalid input syntax for integer: \"health\"","level":"error"}]}
```

**Comment by DylanVolz** [comment]
I have identified these routes to be the ones requiring passthrough to the perl server:

/cdns/dnsseckeys
/cdns/health
/cdns/capacity
/cdns/routing
/cdns/configs
/cdns/domains

**Comment by mitchell852** [comment]
wish i could assign this to @DylanVolz :)

**Comment by rob05c** [comment]
Fixed in #1731
